test-cases/longevity/longevity-custom-nizar.yaml

```diff
@@ -0,0 +1,19 @@
+test_duration: 60
```

Comment:
as bentsi note, please remember to remove this file from the PR/commit

Reply:
I will remove it after resolving all comments